coveo-pyproject/README.md

```diff
@@ -1,240 +1,196 @@
-# pyproject
+# coveo-pyproject
 
-Helps with both the packaging and development sides of python projects backed by poetry.
+Extra magic for poetry-backed projects with CI and batch operations in mind.
 
 
-## Install
+## Use cases
 
-We strongly recommend using `pipx` to wrap this into a nice little space:
+- You don't want to spend time writing the setup/test/report/publish/tag CI workflow
+- You want to work with multiple python projects in a single repository
+- You need a way to download locked dependencies and install them offline later
 
-```
-pip install pipx --user
-pipx install coveo-pyproject
-```
-
-## Commands
-
-`pyproject <command>`
-
-The `pyproject` entrypoint has a bunch of commands designed to interact with `pyproject.toml` and `poetry`.
-
-It's designed to simplify common operations over multiple sub projects.
-
-Some commands can target "all" projects in a repository. What "all" means will depend on your context:
-- Within a git repository, "all" means everything from the first git parent.
-- Outside of the git context, "all" is the current directory, recursive.
-
-### build some-project
-This command outputs the artifacts of your project into a folder, so that it can be distributed based on its `poetry.lock`
-metadata.
-
-This is different from publishing to a `pypi` server, where distribution is based on the _constraints_ you placed
-in `pyproject.toml`.
-
-A nice bonus is that the folder does not require poetry or coveo-pyproject to be installed, so it helps keeping production
-systems as small and specific as they should be.
-
-To install from the artifacts folder: 
-`pip install --no-index --find-links /path/to/.wheels <package-name>`
-
-Without a project name specified, all projects will be built.
-
-Scroll down to the poetry gotchas for a more detailed use case and why this command exists!
 
+# Features
 
-### check-outdated
-Ensure that all files in the repo are up to date and return an error if something's outdated.
+## poetry on steroids / offline distribution
+- Orchestrates several isolated poetry projects in one repo
+- Supports unpublished, local-only libraries and projects
+- Generates offline installation package from lock files
 
-A file is out of date if:
-- the `pyproject.toml` changed but not the `poetry.lock`
-- a `pydev` project's dev requirements changed and `pull-dev-requirements` was not called.
+## developer tools
+- Act on python projects in a git repo by name (path is irrelevant)
```

Comment:
Tu veux dire quoi par "Act on"? Je suis pas sûr non plus si ça devrait pas être "Acts on".

Reply:
"Perform operations on" genre. C'est "Act" dans le sens (you need|want to) act on projects...

Reply:
Ok I see. Le changement de nomenclature d'un item à l'autre dans la liste est un peu confusant, donc j'étais pas sûr. Je pense que ça va comme ça quand même.

Reply:
je vais juste l'enlever lui.